matrix/matrix_operation.py

```diff
@@ -151,7 +155,8 @@ def _shape(matrix: List[list]) -> list:
     return list((len(matrix), len(matrix[0])))
 
 
-def _verify_matrix_sizes(matrix_a: List[list], matrix_b: List[list]) -> Tuple[list]:
+def _verify_matrix_sizes(
+        matrix_a: List[list], matrix_b: List[list]) -> Tuple[list]:
     shape = _shape(matrix_a)
```

Comment:
Don't lines 159 and 160 need to have identical indentation?

Reply:
fixed.

Reply:
E125 continuation line with same indent as next logical line.

Reply:
Line 159